PTV-1165 - add Prediction Class to the download

landml opened the pull request:
@cshenry, @JamesJeffryes, @samseaver  - Add Prediction Class to the download. Also rename columns to match data browser. Just making it easier for users to find the data they are looking for.

landml committed:
PTV-1165 - add Prediction Class to the download and rename columns to match data browser

JamesJeffryes commented on lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm:
```diff
@@ -73,10 +73,11 @@ sub export {
     my $self = shift;
 	my $args = Bio::KBase::ObjectAPI::utilities::args([], {file => 0,path => undef}, @_);
-	my $output = ["geneko\tmediaws\tmedia\taddtlCpd\taddtlCpdBounds\tcustomReactionBounds\tgrowth\tsimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions"];
+	my  $output = ["Growth condition\tGenekKO\tmediaws\tAddtional Compounds\taddtlCpdBounds\tcustomReactionBounds\tObserved Normal growth\tSimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions\tPrediction class"];
```
We should make this consistently CamelCase or Title Case

JamesJeffryes replied:
Pretty sure there's no reimport of the data so I'd lean toward the latter

landml replied:
My objective was to match the data viewer for all fields that they have in common.

landml replied:
I can change them all to Title case if you want.

JamesJeffryes commented:
Hmm, I think I'd rather it be internally consistent than match the viewer
(we should fix the viewer though)

On Tue, Nov 27, 2018 at 10:38 AM Miriam Land <notifications@github.com>
wrote:

> *@landml* commented on this pull request.
> ------------------------------
>
> In lib/Bio/KBase/ObjectAPI/KBasePhenotypes/PhenotypeSimulationSet.pm
> <https://github.com/cshenry/fba_tools/pull/82#discussion_r236747992>:
>
> > @@ -73,10 +73,11 @@ sub export {
>  sub printTSV {
>      my $self = shift;
>  	my $args = Bio::KBase::ObjectAPI::utilities::args([], {file => 0,path => undef}, @_);
> -	my $output = ["geneko\tmediaws\tmedia\taddtlCpd\taddtlCpdBounds\tcustomReactionBounds\tgrowth\tsimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions"];
> +	my  $output = ["Growth condition\tGenekKO\tmediaws\tAddtional Compounds\taddtlCpdBounds\tcustomReactionBounds\tObserved Normal growth\tSimulated growth\tsimulated growth fraction\tgapfilled reaction count\tgapfilled reactions\tPrediction class"];
>
> I can change them all to Title case if you want.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cshenry/fba_tools/pull/82#discussion_r236747992>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AIGfOUCIpJXvZ5zEQAtyZa4_bBNNKvtuks5uzWqagaJpZM4Y1uH5>
> .
>

landml commented:
I'm working on PTV-1165 which only asks for the prediction class in the download. I moved and renamed the 'media' column because up to this point, the user was looking as 'Growth conditions' as the first column. When things get renamed and moved around it is harder to line them up with what you saw before.

Just let me know what you think I should do to move this forward.

landml committed:
Change the title lines to Title case

landml commented:
@JamesJeffryes - I've changed the column heading to be a consistent Title case. The public ticket that this goes with is PUBLIC-571.

JamesJeffryes reviewed (approved):
Looks OK to me. Got the 👍 from Chris too.

landml commented:
Super. Just need to get it released.
